XmlDocConflictResolver/ConflictChecker.cs

```diff
@@ -0,0 +1,323 @@
+
+                    if (xDoc != null)
+                    {
+                        DocsComments.LoadDocsFile(xDoc, docsFile.FullName);
+                    }
+                }
+
+                // Enumerates all XML files.
+                //foreach (FileInfo fileInfo in DocsComments.EnumerateFiles())
+                //{
+                //    XDocument? xDoc = null;
+                //    Encoding? encoding = null;
+                //    try
+                //    {
+                //        var utf8NoBom = new UTF8Encoding(encoderShouldEmitUTF8Identifier: false);
+                //        var utf8Bom = new UTF8Encoding(encoderShouldEmitUTF8Identifier: true);
+                //        using (StreamReader sr = new(fileInfo.FullName, utf8NoBom, detectEncodingFromByteOrderMarks: true))
+                //        {
+                //            xDoc = XDocument.Load(sr);
+                //            encoding = sr.CurrentEncoding;
+                //        }
+                //    }
+                //    catch (Exception ex)
+                //    {
+                //        Log.Error($"Failed to load '{fileInfo.FullName}'. {ex}");
+                //    }
+
+                //    if (xDoc != null && encoding != null)
+                //    {
+                //        DocsComments.LoadDocsFile(xDoc, fileInfo.FullName, encoding);
+                //    }
+                //}
+                //Log.Success("Finished looking for Docs xml files.");
+                //Log.Line();
+            }
+        }
+
+        internal void Start()
+        {
+            if (!IntelliSenseXmlComments.Members.Any())
+            {
+                Log.Error("No IntelliSense XML comments found.");
+                return;
+            }
+
+            if (!DocsComments.Types.Any())
+            {
+                Log.Error("No docs type APIs found.");
+                return;
+            }
+
+            InsertConflictingText();
+        }
+
+        private void InsertConflictingText()
+        {
+            Log.Info("Looking for IntelliSense xml comments that differ from the docs...");
+
+            foreach (IntelliSenseXmlMember member in IntelliSenseXmlComments.Members.Values)
+            {
+                CheckForConflictingTextForMember(member);
+            }
+        }
+
+        private void CheckForConflictingTextForMember(IntelliSenseXmlMember ixmlMember)
+        {
+            bool foundDocsApi;
+            IDocsAPI? ecmaxmlApi;
+
+            // Find docs type or member.
+            if (ixmlMember.IsType())
+                foundDocsApi = DocsComments.Types.TryGetValue(ixmlMember.Name, out ecmaxmlApi);
+            else
+                foundDocsApi = DocsComments.Members.TryGetValue(ixmlMember.Name, out ecmaxmlApi);
+
+            if (foundDocsApi && ecmaxmlApi != null)
+            {
+                if (!ecmaxmlApi.Summary.IsDocsEmpty() &&
+                    String.Compare(ecmaxmlApi.Summary, ixmlMember.Summary) != 0)
```

Comment:
```suggestion
                    string.Compare(ecmaxmlApi.Summary, ixmlMember.Summary) != 0)
```

Try to use the in-built language aliases, i.e.; instead of `String` prefer `string`. See https://blog.paranoidcoding.org/2019/04/08/string-vs-String-is-not-about-style.html